client/src/main/java/io/confluent/kafka/schemaregistry/avro/AvroSchema.java

```diff
@@ -189,6 +197,26 @@ public AvroSchema copy(Metadata metadata, RuleSet ruleSet) {
     );
   }
 
+  @Override
+  public ParsedSchema copy(Map<String, Set<String>> tagsToAdd,
+                           Map<String, Set<String>> tagsToRemove) {
+    AvroSchema schemaCopy = this.copy();
+    JsonNode original;
+    try {
+      original = jsonMapper.readTree(schemaCopy.canonicalString());
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException(e);
+    }
+    modifyFieldLevelTags(original, tagsToAdd, tagsToRemove);
+    return new AvroSchema(original.toString(),
+      schemaCopy.references(),
+      schemaCopy.resolvedReferences(),
+      schemaCopy.metadata(),
+      schemaCopy.ruleSet(),
+      -1,
```

Comment:
Should also keep the version if it was set, `schemaCopy.version()`

Reply:
Should we keep the original version, or 1 more than the original?

Reply:
I think keeping the original version makes sense since it's consistent with the other `copy()` method. Fixed.